ui/analyse/src/study/chapterNewForm.ts

```diff
@@ -73,11 +73,13 @@ export function ctrl(
   }
 
   function open() {
+    cancelCurrentTour();
```

Comment:
does this mean the tour script will be loaded every time someone opens the new chapter form, even if they don't actually use the tour? I'd rather avoid that. Maybe you can use `lichess.pubsub.emit('tour.stop')` instead and listen for it in the tour